tests/test_manifest.py

```diff
@@ -0,0 +1,46 @@
+"""Test ltdmason.manifest."""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+from builtins import *  # NOQA
+from future.standard_library import install_aliases
+install_aliases()
+
+import pkg_resources
+import pytest
```

Comment:
It would be nice if SQuaRE had a coding standard then. There was the RFC on Python 3 for SQuaRE and I take PEP8 as a given.